iCub/conf_icub3/gazebo_icub_left_arm_no_hand.ini

```diff
@@ -49,6 +49,7 @@ stictionUp    0.0   0.0   0.0   0.0   0.0   0.0   0.0
 stictionDwn   0.0   0.0   0.0   0.0   0.0   0.0   0.0
 
 [VELOCITY_CONTROL]
+velocityControlImplementationType integrator_and_position_pid
```

Comment:
Files in the `iCub` directory are generated automatically. Can you copy these files in `conf_manual/iCubGazeboV3` or similar or do this modifications in https://github.com/robotology/icub-models-generator ?

Reply:
Thanks. Since `icub-models-generator` doesn't have corresponding files, and not to open two PRs, I just copied the files into  `conf_manual/iCubGazeboV3_visuomanip` as per your suggestion. See the commit 8dc83910af32de9b147c09c82a7a538947a0465c